### Задача 1.  
Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.  
$Решение:$  
Для определения мат. ожидания и дисперсии случайной непрерывной величины , на отрезке [a,b], формулы:  
${Матем. ожидание} : M(A)= \frac{a+b}{2}\\{Дисперсия: }D(A)= \frac{(b-a)^2}{12}$

In [17]:
def math_exp_and_variance(a,b):
    return f'На промежутке ({a};{b}]\nСреднее значение: М(А) = {(a+b)/2: .2f}\nДисперсия: D(A) = {((b-a)**2)/12: .2f}'
print(math_exp_and_variance(200,800))

На промежутке (200;800]
Среднее значение: М(А) =  500.00
Дисперсия: D(A) =  30000.00


$ Ответ :$   
$ M(A) = 500.00$  
$ D(A) = 30000.00$


### Задача 2.  
О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.  
$Решение:$  
Величину В находящуюся на отрезке (a,b), дисперсия случайной непрерывной равномерно распределенной находим по формуле:  
${Дисперсия}: D(A)= \frac{(b-a)^2}{12}$ Значения по условию: $Дисперсия$ $D(B-a)=0.2$ и левая граница отрезка $a=0.5$  
 Правая граница $b$ величины $B$ по формуле: $ D(B-b)\frac{(b-0.5)^2}{12}=0.2 b-0.5=\sqrt{2.4} b=0.5+\sqrt{2.4}$,  
 Среднее значение по формуле: $ M(B)=\frac{a+b}{2}$

In [18]:
b=0.5+2.4**(1/2)
print(f'Правая граница распределения величины В-b = {b: .3f}\n'
      f'Среднее значение В на промежутке (0.5; {b: .3f}) M(B) = {(b+0.5)/2: .3f}')

Правая граница распределения величины В-b =  2.049
Среднее значение В на промежутке (0.5;  2.049) M(B) =  1.275


$Ответ:$
Правая граница распределения: $b=2.049$
Среднее значение: $M(B)=1.275$

### Задача 3.  
Непрерывная случайная величина X распределена нормально и задана плотностью распределения: $f(x)=\frac{1}{4\cdot\sqrt{2\pi}}\cdot e^{-\frac{(x+2)^2}{32}}$    
Найдите:  
а). M(X)  
б). D(X)  
в). $\displaystyle\sigma$(X) (среднее квадратичное отклонение)  
Фрмула плотности нормального распределения:  
$f(x)=\frac{1}{\sigma\sqrt{2\pi}}\cdot e^{-\frac{(x-a)^2}{2\sigma^2}}$  
$M(X)=a=-2\sigma = 4D(X)=\sigma^2=16$  
      
$Ответ:$  
$а). M(X)=-2;$  
$б). D(X)=16;$  
$в). \sigma=4$

Задача 4. Рост взрослого населения города X имеет нормальное распределение, причем, средний рост равен 174 см, а среднее квадратическое отклонение равно 8 см. посчитайте, какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
1. больше 182 см?
2. больше 190 см?
3. от 166 см до 190 см?
4. от 166 см до 182 см?
5. от 158 см до 190 см?
6. не выше 150 см или не ниже 190 см?
7. не выше 150 см или не ниже 198 см?
8. ниже 166 см?

Задачу можно решить двумя способами: без использования сторонних библиотек (numpy, scipy, pandas и пр.), а затем проверить себя с помощью встроенных функций